tut_rule/tut.bzl

```diff
@@ -0,0 +1,35 @@
+load("//scala:scala.bzl", "scala_binary")
+
+def tut_repositories():
+  native.maven_server(
+    name = "tut_repositories_maven_server",
+    url = "https://dl.bintray.com/tpolecat/maven/",
+  )
+
+  native.maven_jar(
+    name = "org_tpolecat_tut_core",
```

Comment:
this might be namespaced enough -- but should we namespace these with `rules_scala_dep_` or something to sandbox them as best we can?